stdlib/REPL/src/docview.jl

```diff
@@ -299,8 +301,33 @@ function summarize(io::IO, T::DataType, binding::Binding)
     end
 end
 
-function summarize(io::IO, m::Module, binding::Binding)
-    println(io, "No docstring found for module `", m, "`.\n")
+function find_readme(m::Module)::Union{String, Nothing}
+    isnothing(pathof(m)) && return nothing
+    path = dirname(pathof(m))
+    top_path = pkgdir(m)
+    while true
+        for readme in ["README.md", "readme.md", "Readme.md", "ReadMe.md"]
+            readme_path = joinpath(path, readme)
+            if isfile(readme_path)
+                return readme_path
+            end
+        end
+        path == top_path && break # go no further than pkgdir
+        path = dirname(path) # work up through nested modules
+    end
+    return nothing
+end
+function summarize(io::IO, m::Module, binding::Binding; nlines::Int = 200)
+    readme_path = find_readme(m)
+    if !isnothing(readme_path)
+        println(io, "No docstring found for module `", m, "`.\n")
+        readme_lines = readlines(readme_path)
+        println(io, "Displaying the contents of `$(readme_path)`:\n\n")
```

Comment:
1 blank line is enough
```suggestion
        println(io, "Displaying the contents of `$(readme_path)`:\n")
```

Reply:
I don't understand why, but a simple println doesn't actually line break in this display. It requires an extra `\n` hence all the extra `\n`'s (most of which I just took from the original code when moving things around)

Reply:
Turns out to be a bug where there's no way to force a single line break in markdown https://github.com/JuliaLang/julia/issues/16004#issuecomment-754904065
Working around it by printing a single line (which is tidier anyway)